ui/raidboss/data/03-hw/raid/a12s.js

```diff
@@ -213,4 +213,62 @@
       },
     },
   ],
+   timelineReplace: [
+    {
+      'locale': 'cn',
+      'replaceSync': {
+        'The General\'s Wing': '阿里达乌斯之翼',
+        'The General\'s Time': '阿里达乌斯之时',
+        'The General\'s Might': '阿里达乌斯之力',
+        'Judgment Crystal': '审判结晶',
+        'Arrhidaeus\'s Lanner': '阿里达乌斯之速',
+        'Alexander Prime': '至尊亚历山大',
+        '(?<! )Alexander(?! )': '亚历山大',
+        'is no longer sealed': '不再封闭',
+      },
+      'replaceText': {
+         'blazing scourge': '白光之鞭',
+        'attack': '攻击',
+        'Will of the Creator': '神明意志',
+        'Unknown Ability': 'Unknown Ability',
+        'timegate(?!active)': '时空门',
+        'Tetrashatter': '结晶破碎',
+        'Temporal Stasis': '时间停止',
+        'Smash': '粉碎',
+        'Radiant Sacrament': '拜火',
+        '(?<! Radiant)Sacrament': '十字',
+        'Punishing Heat': '惩戒射线',
+        'Plaint of Surety': '誓约神判',
+        'Plaint of Solidarity': '连带神判',
+        'Plaint of Obloquy': '污名神判',
+        'Mega Holy': '百万神圣',
+        'Judgment Crystal': '审判结晶',
+        'Incinerating Heat': '净化射线',
+        'Inception': '时空潜行',
+        'Half Gravity': '小重力', 
```

Comment:
`292:31  error  Trailing spaces not allowed                   no-trailing-spaces`
This error means you should remove space at the end of the line.